Doc/reference/lexical_analysis.rst

```diff
@@ -574,53 +574,65 @@ Standard C.  The recognized escape sequences are:
 +-----------------+---------------------------------+-------+
 | ``\v``          | ASCII Vertical Tab (VT)         |       |
 +-----------------+---------------------------------+-------+
-| ``\ooo``        | Character with octal value      | (1,3) |
+| ``\ooo``        | Character with octal value      | (2,4) |
 |                 | *ooo*                           |       |
 +-----------------+---------------------------------+-------+
-| ``\xhh``        | Character with hex value *hh*   | (2,3) |
+| ``\xhh``        | Character with hex value *hh*   | (3,4) |
 +-----------------+---------------------------------+-------+
 
 Escape sequences only recognized in string literals are:
 
 +-----------------+---------------------------------+-------+
 | Escape Sequence | Meaning                         | Notes |
 +=================+=================================+=======+
-| ``\N{name}``    | Character named *name* in the   | \(4)  |
+| ``\N{name}``    | Character named *name* in the   | \(5)  |
 |                 | Unicode database                |       |
 +-----------------+---------------------------------+-------+
-| ``\uxxxx``      | Character with 16-bit hex value | \(5)  |
+| ``\uxxxx``      | Character with 16-bit hex value | \(6)  |
 |                 | *xxxx*                          |       |
 +-----------------+---------------------------------+-------+
-| ``\Uxxxxxxxx``  | Character with 32-bit hex value | \(6)  |
+| ``\Uxxxxxxxx``  | Character with 32-bit hex value | \(7)  |
 |                 | *xxxxxxxx*                      |       |
 +-----------------+---------------------------------+-------+
 
 Notes:
 
 (1)
+   A backslash can be added at the end of a line to ignore the newline::
+
+      >>> 'This string will not include \
+      ... backslashes or newline characters.'
+      'This string will not include backslashes or newline characters.'
+
+   The same result can be achieved by using parentheses and :ref:`string
+   literals concatenation <string-concatenation>` or :ref:`triple-quoted
+   strings <strings>`.
```

Comment:
```suggestion
   The same result can be achieved using :ref:`triple-quoted strings <strings>`,
   or parentheses and :ref:`string literal concatenation <string-concatenation>`.
```

* Avoid ambiguous syntax (it currently can be equally read as "The same result can be achieved by using parenthesis and (string literal concatenation or triple-quoted strings.")
* Use correct concept name/link title (string _literal_, not _literals_, concatenation)